crates/ty_server/tests/e2e/main.rs

```diff
@@ -735,10 +756,16 @@ impl TestServerBuilder {
         Ok(Self {
             workspaces: Vec::new(),
             test_context: TestContext::new()?,
+            global_options: None,
             client_capabilities,
         })
     }
 
+    pub(crate) fn with_global_options(mut self, options: ClientOptions) -> Self {
+        self.global_options = Some(options);
+        self
+    }
```

Comment:
I think we should use `initialization_options` instead of `global_options` which is what this is from a client's perspective.

Reply:
Would that require that the test specify the entire `initialization_options`? Like, with `{ "settings": global_options }`? 


I'd prefer if we can avoid this. It took me longer than it should have to figure out the right structure of our `initialization_options`.

Reply:
> Would that require that the test specify the entire `initialization_options`? Like, with `{ "settings": global_options }`?

No, I just want to indicate that these are the initialization options because I'd find global options a bit confusing from a client perspective.

Reply:
I can rename it and I just saw that this will take `InitializationOptions` in your settings pr